examples/api-samples/src/browser/sample-view.tsx

```diff
@@ -0,0 +1,43 @@
+/********************************************************************************
+ * Copyright (C) 2019 Arm and others.
```

Comment:
2019? ARM? :) Could you check other new files that it points to 2020 and a proper entity, either you or your organisation

Reply:
Oops, I didn't even read it, I just saw the error for it and copy/pasted the one in the same directory :). So I should change it to something like `Copyright (C) 2020 TORO Limited and others.`?

Reply:
If you made this contribution as part of your work at TORO, yes. Otherwise just use your name.